mandatory/4-tax.js

```diff
@@ -5,8 +5,13 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
 
+const TAX_RATE = 0.2;
```

Comment:
Is there any special meaning to capitalize this variable particularly?
If no, I would suggest follow the camelcase guideline in naming.